R/fa.R

```diff
@@ -191,3 +220,17 @@ fa <- function(name,
 
   svg
 }
+
+get_length_value_unit <- function(css_length) {
```

Comment:
This function will allow malformed values to get through. I suggest something more restrictive (and simpler), like this:

```R
get_length_value_unit <- function(css_length) {
  if (!grepl("^[0-9]+[a-z]+$", css_length)) {
    stop(" ... ")
  }
  list(
    value = as.numeric(sub("[a-z]+$", "", css_length)),
    unit = sub("^[0-9]+", "", css_length)
  )
}
```

Reply:
Thanks, and do we error if no units are provided?

Reply:
You could also check if `unit` is a valid CSS unit type and throw an error if not.

Reply:
Yes, we do. Disregard that.

Reply:
Now done. Thanks.